src/SFA.DAS.PAS.Account.Api/Controllers/EmailController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+using SFA.DAS.PAS.Account.Api.Attributes;
+using SFA.DAS.PAS.Account.Api.Orchestrator;
+using SFA.DAS.PAS.Account.Api.Types;
+using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
+
+namespace SFA.DAS.PAS.Account.Api.Controllers
+{
+    [RoutePrefix("api/email")]
+    public class EmailController : ApiController
+    {
+        private readonly EmailOrchestrator _emailOrchestrator;
+
+        private readonly IProviderCommitmentsLogger _logger;
+
+        public EmailController(EmailOrchestrator emailOrchestrator, IProviderCommitmentsLogger logger)
+        {
+            _emailOrchestrator = emailOrchestrator;
+            _logger = logger;
+        }
+
+        [Route("{ukprn}/send")]
+        [HttpPost]
+        [ApiAuthorize(Roles = "ReadAccountUsers")]
+        public async Task<IHttpActionResult> SendEmailToAllProviderRecipients(long ukprn, ProviderEmailRequest request)
```

Comment:
Trivial point, but could we rename ukprn to providerId throughout, as this is what we more-or-less consistently call it elsewhere.

Reply:
@tom-gough oh also, strictly speaking, I believe providerId/ukprn is an integer, but this may cause you more pain than it is worth dealing with (and this is not more-or-less consistent)

Reply:
I would prefer not to due to confusion with accountProviderId in the calling code